### DISCUSSION TITLE
Download Failed. Created a ticket as per the instructions

### DISCUSSION THREAD

**Discussion by gopikrishnareddyg** [aclap-dev/video-downloadhelper#600] [Help and Q&A]
### Your question

{
  "version": "9.0.1.13",
  "target": "mozilla",
  "channel": "stable",
  "page_url": "https://drtalks.com/peptide-summit-2024/free-access-day-4/",
  "error": {
    "error": "coapp_failure",
    "details": "SideDownload error: : [tcp @ 00000236d95bc640] Failed to resolve hostname manifest.prod.boltdns.net: The name does not resolve for the supplied parameters\r\n[in#0 @ 00000236d7de8840] Error opening input: I/O error\r\nError opening input file https://manifest.prod.boltdns.net/manifest/v1/hls/v4/clear/6314452011001/1f1a1155-02e3-4b13-bb85-03aeb3e07c65/6fa3fb8d-8f46-4f4f-b598-3774d2c56d12/6s/rendition.m3u8?fastly_token=NjY1MDkyZTRfODQwY2VkMjdjOGQ1NmQ5MGY5NGZmZWEzNGFiNTAxYzA1YjMxYmVkZGNjMWYxY2NmNGEyODJhOTQyOTFlYWY5Mg%3D%3D.\r\nError opening input files: I/O error\r\n",
    "id": "coapp_failure_e343a2d2-3cd9-46eb-9aa9-9d7d2b019ac7",
    "downloadable_id": "downloadable_4926310781034270"
  }
}

### Addon & CoApp version and details

version: 9.0.1.13
target: mozilla
channel: stable
lang: en-US
coapp: {"found":true,"path":"C:\\Programs\\DownloadHelperCoApp\\vdhcoapp.exe","version":"2.0.19","new_version":false}
license: {"unset":true}
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:126.0) Gecko/20100101 Firefox/126.0
{
    "used_sidebar_button": true,
    "used_popup_button": true,
    "download_directory": "C:\\knowledge\\health\\peptide-summit-2024DrTalks",
    "view_options": {
        "all_tabs": false,
        "low_quality": false,
        "sort_by_status": true,
        "sort_reverse": false,
        "show_button_clean": true,
        "show_button_clean_all": false,
        "show_button_convert_local": false,
        "hide_downloaded": false
    },
    "use_sidebar": false,
    "open_count_store": 9
}

**Comment by paulrouget** [comment]
Thank you! Do you use a proxy?

**Reply by gopikrishnareddyg** [reply]
Nope. By the way, retry worked.
